sdk/typescript/package.json

```diff
@@ -51,5 +51,8 @@
     "tsdx": "^0.14.1",
     "tslib": "^2.3.1",
     "typescript": "^4.6.3"
+  },
+  "resolutions": {
+    "tsdx/**/node-notifier": "10.0.0"
```

Comment:
Nit: would it be wise to force to 8.0.x to avoid possible incompatibilities?

Reply:
The advice was to upgrade to 10.0.0, perhaps because this website (https://snyk.io/vuln/npm:node-notifier) suggests v8 and v9 had similar security issues.